department-of-veterans-affairs/caseflow
# Added scheduled hearings list back to ScheduleVeteran component

### Description
Adds the scheduled hearing list back to the ScheduleVeteran component to display the filled hearing time slots when scheduling

### Acceptance Criteria

### Testing Plan
1. Shadow user `BVASYELLOW` http://localhost:3000/test/users
2. Navigate to schedule veterans and choose an RO from the list: http://localhost:3000/hearings/schedule/assign
3. Navigate to the AMA Waiting tab and choose one of the veterans
4. On the Case Details page choose the "Schedule Veteran" option from the task actions
5. Ensure that the existing hearings display in the time slots
